File: Python/Manual 02/Question03.py
obtainedMarks = float(input("Enter Obtained Marks: "))
totalMarks = float(input("Enter Total Marks: "))

if totalMarks < obtainedMarks or obtainedMarks < 0 or totalMarks < 0:
    print("Enter Valid Data")
else:
    percentage = obtainedMarks / totalMarks

    if percentage >= 0.90:
        print("Grade A")
    elif percentage >= 0.80:
        print("Grade B")
    elif percentage >= 0.70:
        print("Grade C")
    elif percentage >= 0.60:
        print("Grade D")
    elif percentage >= 0.40:
        print("Grade E")
    else:
        print("Grade F")

File: C++/Manual 02/Question03.cpp
#include <iostream>
using namespace std;

int main()
{
    int tmarks, gmarks, rslt;
    cout << "Input Gained Marks: ";
    cin >> gmarks;
    cout << "Input Total Marks: ";
    cin >> tmarks;
    rslt = (gmarks * 100) / tmarks;
    cout << "Your Percentage is: " << rslt << "%" << endl;

    switch (rslt >= 90)
    {
    case 1:
        cout << "Your Grade is A!";
        break;
    }
    switch (rslt >= 80 && rslt <= 89)
    {
    case 1:
        cout << "Your Grade is B!";
        break;
    }
    switch (rslt >= 70 && rslt <= 79)
    {
    case 1:
        cout << "Your Grade is C!";
        break;
    }
    switch (rslt >= 60 && rslt <= 69)
    {
    case 1:
        cout << "Your Grade is D!";
        break;
    }
    switch (rslt >= 40 && rslt <= 59)
    {
    case 1:
        cout << "Your Grade is E!";
        break;
    }
    switch (rslt < 40)
    {
    case 1:
        cout << "Your Grade is F!";
        break;
    }

    return 0;
}
